packages/runtime/container-runtime/src/batchTracker.ts

```diff
@@ -35,7 +35,7 @@ export class BatchTracker {
                 this.startBatchSequenceNumber !== undefined && this.batchProcessingStartTimeStamp !== undefined,
                 0x2ba /* "batchBegin must fire before batchEnd" */);
 
-            const length = message.sequenceNumber - this.startBatchSequenceNumber + 1;
+            const length = message.clientSequenceNumber - this.startBatchSequenceNumber + 1;
```

Comment:
I think this change is noop. Curious, why do you want to change it?

Reply:
synced offline